### ISSUE TITLE
debian 11 duplaceted rules after restart

### ISSUE THREAD

**Issue by thefiredragon** [greenpau/cni-plugins#18]
after stop start a container rules added twice which prevent forwarding
after stopping a container I need to do  nft flush ruleset and start container.

**Comment by AlekseyMelikov** [comment]
Yes, I see a similar problem in Debian 11. I'll try to figure it out later.

**Comment by vrga** [comment]
the issue seems to be incomplete cleanup of rules it creates.

**Comment by greenpau** [comment]
@thefiredragon , @AlekseyMelikov , @vrga , PRs are welcomed!
